Assets/Plugins/LeapMotion/Core/Plugins/LeapCSharp/Connection.cs

```diff
@@ -888,4 +902,12 @@ private void handleDeviceStatusEvent(ref LEAP_DEVICE_STATUS_CHANGE_EVENT statusE
       _lastResult = result;
     }
   }
+
+  public UInt64 SetTrackingMode(eLeapTrackingMode mode) {
```

Comment:
In C# you cannot have free functions inside a namespace - these will need to go into a class.

Reply:
It was indeed not in the Connection class (move one lined above)